### PR TITLE
Update data.gov.uk manual pages for post-upgrade

### PR DESCRIPTION
**Pull request by brucebolt**
There will be lots of changes to how DGU operates after the CKAN upgrade/migration has been completed.  This PR reflects those changes.

Trello card: https://trello.com/c/DYlns7Va/99-review-and-update-dgu-dev-docs

### CODE REVIEW

File: source/manual/data-gov-uk-supporting-ckan.html.md [hunk 1]
```diff
@@ -4,108 +4,41 @@ title: Supporting CKAN
...
-```
-sudo supervisorctl restart ckan_fetch_consumer_dgu
-sudo supervisorctl restart ckan_gather_consumer_dgu
-```
-
-If your change is not showing up on the website, it may be necessary to restart Apache:
-
-```
-sudo service apache2 restart
+sudo -u deploy govuk_setenv ckan venv/bin/paster [COMMAND] -c /var/ckan/ckan.ini
```
[the remark this answers is not in the document]
Comment (elliotcm) [reply source/manual/data-gov-uk-supporting-ckan.html.md]: (answered for myself: no, not unless we do it in some other way)

File: config/tech-docs.yml [hunk 2]
```diff
@@ -37,6 +37,7 @@ redirects:
   /manual/create-page.html: /manual/docs-style-guide.html
   /manual/creating-a-new-environment.html: /manual.html
   /manual/creating-pages.html: /manual/docs-style-guide.html
+  /manual/data-gov-uk-backups.html: /manual/postgresql.html
```
[the remark this answers is not in the document]
Comment (elliotcm) [reply config/tech-docs.yml]: Yeah.  You mentioned it would take quite a few hours to rebuild the index?

[the remark this answers is not in the document]
Comment (brucebolt) [reply config/tech-docs.yml]: https://trello.com/c/k8F6imkz/141-add-backups-for-solr-index

File: source/manual/data-gov-uk-supporting-ckan.html.md [hunk 3]
```diff
@@ -298,7 +230,7 @@ paster --plugin=ckanext-harvest harvester jobs -c $CKAN_INI
 It may be faster to run a SQL query to get the ID of a specific harvest job.
 
 ```
-psql ckan -c "SELECT id FROM harvest_source WHERE name = '[NAME]'"
+psql ckan_production -c "SELECT id FROM harvest_source WHERE name = '[NAME]'"
```
Comment (brucebolt) [inline source/manual/data-gov-uk-supporting-ckan.html.md]: Note to self: update this since the database is no longer on the same machine

**Pull request merged**